### PR TITLE
SPI-00001: No implementation class load from SPI `org.apache.shardingsphere.sqltranslator.spi.SQLTranslator`

### PR DESCRIPTION
**Pull request by cheese8**
Changes proposed in this pull request:
-

### CODE REVIEW

**Comment by misselvexu** [comment]
The trunk branch build failed ,and when this PR will be merged ? 😔

**Comment by cheese8** [comment]
> The trunk branch build failed ,and when this PR will be merged ? 😔

Yes, I am try to fix the trunk.

File: shardingsphere-kernel/shardingsphere-sql-translator/shardingsphere-sql-translator-core/pom.xml [hunk 1]
```diff
@@ -33,5 +33,10 @@
             <artifactId>shardingsphere-sql-translator-api</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.shardingsphere</groupId>
+            <artifactId>shardingsphere-sql-translator-native-provider</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```
Comment (terrymanu) [inline shardingsphere-kernel/shardingsphere-sql-translator/shardingsphere-sql-translator-core/pom.xml:40]: Core module should not aware the provider modules

**Review by terrymanu — CHANGES_REQUESTED** [review]

File: shardingsphere-kernel/shardingsphere-sql-translator/shardingsphere-sql-translator-core/pom.xml (hunk 1 above)
Comment (cheese8) [reply shardingsphere-kernel/shardingsphere-sql-translator/shardingsphere-sql-translator-core/pom.xml:40]: move this import to shardingsphere-infra-rewrite module? @terrymanu

**Pull request closed without merging**